icaruscode/TPC/Calorimetry/NormalizeYZ_tool.cc

```diff
@@ -132,18 +132,18 @@ const icarus::calo::NormalizeYZ::ScaleInfo& icarus::calo::NormalizeYZ::GetScaleI
       thisscale = scale;
       found_scale_t0 = true;
 
-      if (fVerbose) std::cout << "NormalizeYZ Tool -- Found prior matching T0 from timestamp: " << scale_pair.first << std::endl;
+      if (fVerbose) std::cout << "NormalizeYZ Tool -- Found prior matching T0 from run: " << scale_pair.first << std::endl;
 
       break;
     }
   }
 
   if (found_scale_t0) {
-    fScaleInfos[timestamp] = thisscale;
-    return fScaleInfos.at(timestamp);
+    fScaleInfos[run] = thisscale;
+    return fScaleInfos.at(run);
```

Comment:
Or:
```suggestion
    return fScaleInfos[run] = thisscale;
```
(but it may confuse people)